.promu.yml

```diff
@@ -1,6 +1,5 @@
 go:
   version: 1.9.2
-verbose: false
```

Comment:
I don't think we need this change, but not a blocker, I'll follow up with a PR cleaning up the promu mentions as the Docker build pipeline doesn't use this for building.